packages/dd-trace/src/encode/agentless-ci-visibility.js

```diff
@@ -0,0 +1,245 @@
+'use strict'
+const tracerVersion = require('../../lib/version')
+const { truncateSpan, normalizeSpan } = require('./tags-processors')
+const Chunk = require('./chunk')
+const log = require('../log')
+const { AgentEncoder } = require('./0.4')
+
+const ENCODING_VERSION = 1
+
+function formatSpan (span) {
+  return {
+    type: span.type === 'test' ? 'test' : 'span',
```

Comment:
What's `type: span`?

Reply:
for the moment there's only two types of events: tests and spans (everything that is not a test)

Reply:
Is it an explicit type or is it implicit? If it's implicit this could just be left untouched when it's not `test`.

Reply:
it is explicit